rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -248,15 +245,16 @@ components:
       properties:
         name:
           type: string
-          description: >
-            The name of the contact person.
+          description: The name of the contact person.
           minLength: 1
+          maxLength: 255
           example: Bob Patterson
         email:
           type: string
           format: email
-          description: >
-            The email address for the contact person.
+          description: The email address for the contact person.
+          minLength: 1
```

Comment:
❓ Should this be 5? Ex: a@b.c or does the format:email handle that?

Reply:
The format already handles that part.